hs_core/templates/pages/genericresource.html

```diff
@@ -502,6 +502,14 @@ <h4 class="modal-title">Confirm file deletion</h4>
                             </a>
                         </li>
                     {% endif %}
+                    {% if relevant_tools %}
+                        <li class="col-md-3 col-sm-3 col-xs-12">
+                            <a href="#relevantTools" data-toggle="tab">
+                                <span class="glyphicon glyphicon-wrench"></span>
+                                &nbspRelevant Tools
```

Comment:
With this, I mimicked the other preexisting tabs. I added semicolons to mine and replaced "&nbsp" with "&nbsp;" throughout the document. As for switching the effect to padding, @mauriel133 did most of the styling on this page and may be able to make that switch, if necessary, more easily than I.